bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/HierarchicalLedgerManager.java

```diff
@@ -86,6 +86,12 @@ protected long getLedgerId(String ledgerPath) throws IOException {
         return StringUtils.stringToLongHierarchicalLedgerId(hierarchicalPath);
     }
 
+    @Override
+    public boolean isSpecialZnode(String znode) {
```

Comment:
This would probably have to be legacyLM.isSpecialZnode(znode) || longLM.isSpecialZnode(znode) || super.isSpecialZnode(znode);

Reply:
as we discussed, I'm merging all of these isSpecialZnode methods into one method, for the sake of simplicity and clarity.